fuzz.go

```diff
@@ -503,31 +504,65 @@ type int63nPicker interface {
 	Int63n(int64) int64
 }
 
-type charRange struct {
-	first, last rune
+// UnicodeRange is a structure that provides the developer with setting
```

Comment:
I suggest: UnicodeRange describes a sequential range of unicode characters. Last must be numerically greater than First.

Reply:
Fix